packages/labs/grpc_web/BUILD.bazel

```diff
@@ -42,10 +42,11 @@ well_known_protos = [
 
 js_library(
     name = "bootstrap_scripts",
-    srcs = [
+    named_module_srcs = [":google-protobuf-%s__umd" % p for p in well_known_protos],
```

Comment:
this is where you get an error on CI
`ERROR: /home/circleci/rules_nodejs/packages/labs/test/grpc_web/BUILD.bazel:41:1: in srcs attribute of _karma_web_test rule //packages/labs/test/grpc_web:proto_with_deps_test_suite_wrapped_test: '//packages/labs/grpc_web:bootstrap_scripts' does not produce any _karma_web_test srcs files (expected .js). Since this rule was created by the macro 'karma_web_test_suite', the error might have been caused by the macro implementation`

it's suspicious that you moved some of the `deps` to `named_module_srcs` - seems like it would need to be all or none, since they're all __umd targets?

Reply:
👍